frontend/src/components/company/CandidatList.jsx

```diff
@@ -6,6 +6,7 @@ import style from "../../assets/styles/messagePage.module.scss";
 function CandidatList() {
   const { auth } = useOutletContext();
   const [candidates, setCandidates] = useState();
+  const candidate = [candidates];
```

Comment:
no need for new const here, better to rename the state

Reply:
i need it to create an array of objects as the state only recovers objects